src/tests/ov_helpers/ov_snippets_models/src/subgraph_mlp.cpp

```diff
@@ -0,0 +1,283 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "subgraph_mlp.hpp"
+
+#include "snippets/op/subgraph.hpp"
+
+#include "openvino/opsets/opset15.hpp"
+#include "ov_lpt_models/common/builders.hpp"
+#include "ov_lpt_models/common/dequantization_operations.hpp"
+#include "ov_lpt_models/common/fake_quantize_on_data.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+std::shared_ptr<ov::Model> MLPSeqFunction::initOriginal() const {
+    auto A_param = std::make_shared<ov::op::v0::Parameter>(precisions[0], input_shapes[0]);
+    std::shared_ptr<Node> A = A_param;
+    if (precisions[0] != ov::element::f32) {
+        A = std::make_shared<ov::op::v0::Convert>(A, ov::element::f32);
+    }
+    auto b_shape = ov::Shape{static_cast<unsigned long>(input_shapes[0][1].get_length()),
+                             static_cast<unsigned long>(input_shapes[0][1].get_length())};
+    auto b_row = ov::Shape{static_cast<unsigned long>(input_shapes[0][1].get_length())};
+    auto add = std::make_shared<ov::op::v0::Constant>(ov::element::f32,
+                                                      b_row,
+                                                      std::vector<float>{0.1122});
+    std::shared_ptr<Node> current = A;
+
+    for (size_t mm_count = 0; mm_count < num_input_nodes; ++mm_count) {
+        auto B = std::make_shared<ov::op::v0::Constant>(ov::element::f32, b_shape, std::vector<float>{0.1122f + mm_count});
+        current = std::make_shared<ov::op::v0::MatMul>(current, B, false, true);
+        current = std::make_shared<ov::op::v1::Multiply>(current, add);
```

Comment:
`Multiply` shouldn't be in original models: common fp32 model and quantized models (not type relaxed). I think you inserted `Multiply` here because model after LPT has this op after `MatMul` - but this `Multiply` is dequantized op which is appeared after LPT in quantized models

Reply:
Removed `Multiply`